ocs_ci/ocs/resources/pvc.py

```diff
@@ -412,7 +412,10 @@ def create_pvc_snapshot(
     Returns:
         OCS object
     """
+    ocp_version = version.get_semantic_ocp_version_from_config()
     snapshot_data = templating.load_yaml(snap_yaml)
+    if ocp_version >= version.VERSION_4_11:
+        snapshot_data["apiVersion"] = "snapshot.storage.k8s.io/v1"
```

Comment:
if we are creating any volume snapshotclass same change is required for VSC also.

Reply:
The yaml for snapshotclass is used directly in different test cases and functions. A common function to create volumesnapshotclass is not used in different test cases. We will address the change in volumesnapshotclass yaml as part of https://github.com/red-hat-storage/ocs-ci/issues/6050.
The change in the PR will unblock acceptance run